src/Summoner/Template.hs

```diff
@@ -298,24 +298,30 @@ createProjectTemplate ProjectData{..} = Dir (toString repo) $
 
     -- create README template
     readme :: Text
-    readme =
-        [text|
-        # $repo
-
-        [![Hackage]($hackageShield)]($hackageLink)
-        [![$license license](${licenseShield})](${licenseLink})
-        $stackBadges
-        $travisBadge
-        $appVeyorBadge
-
-        $description
-        $endLine
-        |]
+    readme = T.intercalate "\n" $ filter (/="") 
+        [ ("# " <> repo <> "\n")
+        , hackage
+        , licenseBadge
+        , stackBadges
+        , travisBadge
+        , appVeyorBadge
+        , ("\n" <> description)
+        ]
       where
         hackageShield :: Text =
-          "https://img.shields.io/hackage/v/" <> repo <> ".svg"
+            "https://img.shields.io/hackage/v/" <> repo <> ".svg"
         hackageLink :: Text =
-          "https://hackage.haskell.org/package/" <> repo
+            "https://hackage.haskell.org/package/" <> repo
+        hackage :: Text = makeBadge "Hackage" hackageShield hackageLink
+
+        licenseShield :: Text =
+            "https://img.shields.io/badge/license-" <> T.replace "-" "--" license <> "-blue.svg"
+        licenseLink :: Text = memptyIfFalse github $ 
+            "https://github.com/" <> owner <> "/" <> repo <> "/blob/master/LICENSE"
```

Comment:
This is not needed. The license link should always be the one that you wrote in the next line.